crates/nu-command/tests/commands/into_filesize.rs

```diff
@@ -61,3 +61,10 @@ fn into_filesize_negative_filesize() {
 
     assert!(actual.out.contains("-3.0 KiB"));
 }
+
+#[test]
+fn into_negative_filesize() {
+    let actual = nu!("-1 | into filesize");
+
+    assert!(actual.out.contains("-1 B"));
```

Comment:
```suggestion
    let actual = nu!("'-1' | into filesize");

    assert!(actual.out.contains("-1 B"));
```

Same thing here, there's already a negative filesize test above. Thanks!

Reply:
Updated